content/ending-frozen-applications.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Ending Frozen Applications and Desktops
```

Comment:
I don't think this title is confusing at all, but I wonder if "Stopping" or "Closing" would be what users would search for first instead of "Ending".